easybuild/easyconfigs/p/ProbABEL/ProbABEL-0.5.0.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'ConfigureMake'
+
+name = 'ProbABEL'
+version = '0.5.0'
+
+homepage = "http://www.genabel.org/packages/ProbABEL"
+description = """Tool for genome-wide association 
+analysis of imputed genetic data."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['http://www.genabel.org/sites/default/files/software/']
+sources = ['probabel-%(version)s.tar.gz']
+
+dependencies = [
+    ('Eigen', '3.2.6')
+]
+
+configopts = '--disable-latex-doc'
+
+sanity_check_paths = {
+    'files': ['bin/probabel', 'bin/pacoxph',
+	'bin/palinear', 'bin/palogist','etc/probabel_config.cfg.example'],
+    'dirs': ['share'],
+}
+
+moduleclass = 'bio'
+
+## You need to change the file /etc/probabel_config.cfg.example
+## to etc/probabel_config.cfg in the install directory for
+## it to work.
```

Comment:
@boegel : It's just renaming the file, Kenneth.
If you don't do it, running the executables gives an error saying you forgot to rename the file, and won't work. Is there a way I can do the renaming in the Easyconfig itself, so we don't have to leave it to the users to change it manually?